cli/Cargo.toml

```diff
@@ -42,10 +42,10 @@ winres.workspace = true
 [dependencies]
 deno_ast = { workspace = true, features = ["bundler", "cjs", "codegen", "dep_graph", "module_specifier", "proposal", "react", "sourcemap", "transforms", "typescript", "view", "visit"] }
 deno_core = { workspace = true, features = ["include_js_files_for_snapshotting"] }
-deno_doc = "0.57.0"
-deno_emit = "0.16.0"
-deno_graph = "=0.44.3"
-deno_lint = { version = "0.41.0", features = ["docs"] }
+deno_doc = "0.59.0"
+deno_emit = "0.17.0"
+deno_graph = "=0.45.0"
+deno_lint = { version = "0.43.0", features = ["docs"] }
 deno_lockfile.workspace = true
 deno_runtime = { workspace = true, features = ["dont_create_runtime_snapshot", "include_js_files_for_snapshotting"] }
```

Comment:
Unrelated to this PR, but we should probably lift all these definitions to root `Cargo.toml`